utils/build_swift/build_swift/driver_arguments.py

```diff
@@ -644,6 +644,12 @@ def create_argument_parser():
                 '`bootstrapping-with-hostlibs`, `crosscompile`, and '
                 '`crosscompile-with-hostlibs`')
 
+    option('--use-linker', store('use_linker'),
+           choices=['gold', 'lld'],
```

Comment:
Newer versions of the GNU `bfd` linker works on newer platforms now too and IIRC `gold` is going away so I don't think we should block using it anymore.

Reply:
My understanding is that bfd doesn't have a specific name... so people will just get it by not setting the flag.